### PR TITLE
Splash Page Styling

### PR DESCRIPTION
**Pull request by hilakey**
+ Made Beetle Logo float in "air"
+ Rounded submit button
+ added glow on borders for the dropdowns when active(helps user know they can be interacted with)

See here ----> [](url) https://play-user-data-beetle.uc.r.appspot.com/

### CODE REVIEW

**Review by jiaxinz-g — APPROVED** [review]

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -168,14 +169,19 @@ function addReasonInput() {
   reason.setAttribute("name", "reason");
   reason.setAttribute("placeholder", "Enter reason for use.");
   reason.setAttribute("required", "true");  
+  reason.focus();//makes a glowing border on focus/hover
```
Comment (wdcroslen) [inline src/main/webapp/script.js:172]: I really liked the glow when I was testing this out!

File: src/main/webapp/splash.css [hunk 2]
```diff
@@ -64,4 +100,19 @@ input[type=submit] {
...
+
+#logo-bounce {
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+  transform-origin: bottom;
+  animation-name: logo-bounce;
+  animation-timing-function: ease-in-out;
+}
+
+@keyframes logo-bounce {
```
Comment (wdcroslen) [inline src/main/webapp/splash.css:111]: Super cool!

**Review by wdcroslen — APPROVED** [review]

File: src/main/webapp/main-style.css [hunk 3]
```diff
@@ -63,6 +63,17 @@ input{
...
+#submit {
+  background-color:  darkseagreen;
+  border-color: cadetblue;
+  border-style: solid;
+  margin-left: 10px;
+  font-size: 20px;
+  font-style: bold;
+  color: white;
+}
+
```
Comment (sasymer) [inline src/main/webapp/main-style.css]: Could you take this out maybe? I think we changed the colors in a different PR to fit the color scheme better

File: src/main/webapp/script.js (hunk 1 above)
Comment (sasymer) [reply src/main/webapp/script.js:172]: Same!

File: src/main/webapp/splash.css [hunk 4]
```diff
@@ -32,18 +36,42 @@ body {
...
+.select:hover {
+    box-shadow: 0 0 10px #55d1ff;
+}
+
+.select:focus {
+    outline:none;
+}
+
+
+
```
Comment (sasymer) [inline src/main/webapp/splash.css]: Extra space

**Review by sasymer — APPROVED** [review]
So excited to see this feature!!

File: src/main/webapp/index.html [hunk 5]
```diff
@@ -11,7 +11,9 @@
   <body onload="onLoad()">
     <div class="centered">
       <div name="logoImage">
+        <div id="logo-bounce">
         <img src="beetle-black.png" style="width: 100px" alt="logo here">
```
Comment (gagomez) [inline src/main/webapp/index.html]: Let's tab the img one more

**Review by gagomez — APPROVED** [review]

File: src/main/webapp/main-style.css (hunk 3 above)
Comment (hilakey) [reply src/main/webapp/main-style.css]: Done!

File: src/main/webapp/splash.css (hunk 4 above)
Comment (hilakey) [reply src/main/webapp/splash.css]: Done!

File: src/main/webapp/index.html (hunk 5 above)
Comment (hilakey) [reply src/main/webapp/index.html]: Yes just fixed it

**Pull request merged**